ompi/errhandler/errhandler.c

```diff
@@ -484,20 +489,26 @@ static void *ompi_errhandler_event_cb(int fd, int flags, void *context) {
                 continue; /* we are not 'MPI connected' with this proc. */
             }
             assert( !ompi_proc_is_sentinel(proc) );
-            ompi_errhandler_proc_failed_internal(proc, OPAL_ERR_PROC_ABORTED, false);
+            ompi_errhandler_proc_failed_internal(proc, status, false);
         }
         opal_event_del(&event->super);
         free(event);
         return NULL;
+    case PMIX_ERR_LOST_CONNECTION:
```

Comment:
you removed PMIX_ERR_LOST_CONNECTION from the list of codes PMIX will generate. How can it become an event here ?

Reply:
We can still get there through the handler registered on line 423 in instance.c (see that file below) which has no specific event codes, thus is a default/catchall handler. Also for debugging purposes I really want a good way to get a message showing up when these things happen.